src/lib/config/defaultConfig.js

```diff
@@ -166,10 +166,13 @@ export const RECORDS_CONFIG = {
   CIRCULATION: {
     deliveryMethods: {
       PICKUP: {
-        text: 'Pick it up at the library desk',
+        text: "I'd like to pick it up at the library desk",
```

Comment:
To make it consistent with the header we discussed during the demo (which should be "Delivery options:")
```suggestion
        text: "Pick up at the library desk",
```